src/main/resources/db/migration/V1_68__create_utilisation_report.sql

```diff
@@ -0,0 +1,307 @@
+-- Function to return daily utilisation stats for a court between two dates
+CREATE OR REPLACE FUNCTION juror_mod.util_report_daily_summary(p_loc_code text, p_start_date date, p_end_date date)
+ RETURNS TABLE(
+	report_date date,
+	working bigint,
+	sitting bigint,
+	attended bigint,
+	non_attendance bigint,
+	utilisation numeric)
+ LANGUAGE plpgsql
+AS $function$
+begin
+
+return query select
+	report_main.report_date,
+	sum(report_main.working) as working_days,
+	sum(report_main.sitting) as sitting_days,
+	sum(report_main.attended) as attendance_days,
+	sum(report_main.non_attendance) as non_attendance_days,
```

Comment:
this could be calculated here as working_days - attendance_days ... I assume it's being calculated lower down but may be able to remove the non_attendance_days from the util_report_main table function and calculate it once here instead of calculating it earlier then summing it here

Reply:
have updated as suggested